topics/proteomics/tutorials/encyclopedia/tutorial.md

```diff
@@ -0,0 +1,428 @@
+>       - To: **191023JAT12_4_Mix_3_1ug_pOT_30k_390_1010_12.raw**
+>       - Again, the "%20" characters were removed
+>     - The Background FASTA file will be renamed similarly:
+>       - From: **T4_Salmonella_Ecoli_Bacillus_BS_191102.fasta**
+>       - To: **T4_Salmonella_Ecoli_Bacillus_BS_191102.fasta**
+>     - Lastly, the DLIB library is renamed:
+>       - From: **T4_Salmonella_Ecoli_Bacillus_fasta_trypsin_z2_nce33_BS_191102.dlib**
+>       - To: **T4_Salmonella_Ecoli_Bacillus_fasta_trypsin_z2_nce33_BS_191102.dlib**
+> 4. Check that the datatype
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 5. Add to each dataset a tag corresponding to file-type, i.e.
+>    - GPF datasets can be labeled as `#gpf`
+>    - Experimental DIA datasets as `#experimental`
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+> 6. Please create a dataset collection for both the GPF files and the Experimental Design files.
+>
+>    {% snippet faqs/galaxy/collections_build_list.md %}
+>
+{: .hands_on}
+
+# Conversion of file types
+
+msconvert is the first tool in this EncyclopeDIA workflow as before analysis of DIA data may begin, the data files (.RAW format) must be converted to the correct file-type (mzML) from the MS/MS data. Conversion from .raw to .mzML is important because the SearchToLib tool (responsible for generation of the Chromatogram Library), as well as the EncyclopeDIA tool (responsible for analysis of DIA data) require mzmL inputs. As msconvert exists on the Galaxy platform, conversion of files to the necessary type is straightforward, and can be incorporated into the workflow itself as opposed to a separate precursor. Both the GPF DIA raw data and the Experimental DIA raw data are run through msconvert for conversion to mzML for the following steps, creation of the Chromatogram Library and analysis of DIA data through EncyclopeDIA.
+
+In this workflow, msconvert uses dataset collections. The tool will convert each of the data files in the collection from a raw file-type to a mzML file-type. Then, a dataset collection containing the mzML files will be generated as the output from msconvert. msconvert will run twice, as both the GPF raw DIA data as well as the Experimental DIA raw data need to be converted to mzML file-type. Therefore, two outputs will be generated:
+
+- GPF mzML DIA dataset collection
+
+  This output will serve as an input for SearchToLib in generating the Chromatogram Library.
+
+- Experimental mzML DIA dataset collection
+
+  This output will be the DIA data analyzed with EncyclopeDIA.
+
+
+> ### {% icon hands_on %} Hands-on: Conversion of GPF DIA mass spectrometry raw data to mzML data-type.
+>
+> 1. {% tool [msconvert](toolshed.g2.bx.psu.edu/repos/galaxyp/msconvert/msconvert/3.0.19052.1) %} with the following parameters:
+>    - {% icon param-collection %} *"Input unrefined MS data"*: `output` (Input dataset collection)
```

Comment:
what is `output` here? This should be the name of the file or collection they should use as input here (or a description of it). I guess it is one of the collections we made in the last step. It could help if you add instructions in the collection making step to give the collection a name, so that you can use it here